server.js

```diff
@@ -415,8 +419,35 @@ function prStatus (pr, delta, cb) {
                                         pr.contribStatus[username] = "ok";
                                         return cb(null, "ok");
                                     } else {
-                                        pr.contribStatus[username] = "not in group";
-                                        return cb(null, "not in group");
+                                        // we assume that all groups are of the same type
+                                        store.getGroup(repoGroups[0], function(err, group) {
+                                            if (err) return cb(err);
+                                            if (!group) return cb("Unknown group: " + repoGroups[0]);
+                                            if (group.groupType === 'WG') {
+                                              log.info("Looking up for non-participant licensing contribution");
+                                              gh.getRepo(pr, function(err, res) {
+                                                  if (err) return cb(err);
+                                                  w3c.nplc({repoId: res.id, pr: pr.num}).fetch(function(err, w3cnplc) {
+                                                      if (err) {
+                                                          // Non-participant licensing contribution doesn't exist yet
+                                                          pr.contribStatus[username] = "not in group";
+                                                          return cb(null, "not in group");
```

Comment:
shouldn't this read something else than "not in group"? (e.g. "no commitment made" which would cover both the "not in group" and "no nplc made")

Reply:
addressed https://github.com/w3c/ash-nazg/pull/163/commits/3491667e19f54f1d4a3e43e40c05b151d66203a0